tools/src/tester/StoryParser.ts

```diff
@@ -0,0 +1,39 @@
+/*
+* Copyright OpenSearch Contributors
+* SPDX-License-Identifier: Apache-2.0
+*
+* The OpenSearch Contributors require contributions made to
+* this file be licensed under the Apache-2.0 license or a
+* compatible open source license.
+*/
+
+import _ from "lodash";
+import { ChapterRequest, Story } from "./types/story.types";
+import { ParsedChapter, ParsedChapterRequest, ParsedStory } from "./types/parsed_story.types";
+
+export default class StoryParser {
+  static parse(story: Story): ParsedStory {
+    return {
+      ...story,
+      chapters: this.#expand_chapters(story.chapters) as ParsedChapter[],
+      prologues: this.#expand_chapters(story.prologues),
+      epilogues: this.#expand_chapters(story.epilogues)
```

Comment:
We're not testing the `prologues` and `epilogues`, right? Their goal is simply executing the API commands successfully, and are not concerned with different ways of executing that same command. So, I don' think we should support this feature for the set-up and tear-down.

Reply:
Removed.